### DISCUSSION TITLE
请问文件分块失败是什么原因？

### DISCUSSION THREAD

**Discussion by BiFangKNT** [lobehub/lobehub#4058] [Self Hosting] [answered]
如图
![image](https://github.com/user-attachments/assets/1a4f2d76-4001-45f8-8f86-be73242ff9c3)

**Comment by BiFangKNT** [comment]
lobe和minio的日志没有报错

**Reply by BiFangKNT** [reply]
minio没有报错，lobe有

**Comment by BiFangKNT** [comment]
我看过站内那个相关的讨论了，但是我不认为我们的原因相同。我这是自部署的minio，也做了跨域访问，不应该屏蔽我的域名啊
浏览器报的是403
![image](https://github.com/user-attachments/assets/073630f5-17cf-4afa-b04a-0c5073acfd05)

**Reply by arvinxx** [reply]
升级下版本到 1.19.20？ 这个抛错是个 bug

**Reply by BiFangKNT** [reply]
> 升级下版本到 1.19.20？ 这个抛错是个 bug

![image](https://github.com/user-attachments/assets/84a74bfd-228f-4f08-a22e-8df1283a21e6)
已经是最新版本了

**Reply by BiFangKNT** [reply]
> 升级下版本到 1.19.20？ 这个抛错是个 bug

这个403报错应该是因为我顶级域名是.top，这个问题我已经明了了，不过这个讨论下面那两个问题希望大佬能解答一下！

**Reply by BiFangKNT** [reply]
这个403报错应该与我的文件分块失败无关。

**Comment by BiFangKNT** [comment]
403
![image](https://github.com/user-attachments/assets/52db30e3-4794-4172-94ee-e7a1f1c6edbc)
![image](https://github.com/user-attachments/assets/4a91a449-889e-4a0d-b6b5-10650eb74cbb)

**Reply by BiFangKNT** [reply]
此403报错应该与我的文件分块问题无关

**Comment by BiFangKNT** [comment, marked as answer]
等等，这个分块功能只能使用`text-embedding-3-small`这个模型吗？我好像知道是什么原因了🤣

**Reply by BiFangKNT** [reply]
分块所依赖的向量模型能使用别的模型吗？比如glm的`embedding-3`？

**Comment by BiFangKNT** [comment]
再提一个相关的问题：每个文件的首次分块才会报上面的模型相关的错误，尝试再次分块时不会报前者的错误，但是却会报dns错误，请问现在lobe还需要配置dns配置文件映射吗？详见 #3814 中的最新回复

**Reply by BiFangKNT** [reply]
这里提到的两个报错都是lobe容器日志里报的

**Reply by BiFangKNT** [reply]
我在lobe容器内部测试过了，解析没问题的（域名和ip我隐藏了）：
```
nextjs@fb7f6009f7c1:/$ nslookup lobechat.example.top
Server: 223.5.5.5
Address: 223.5.5.5:53

Non-authoritative answer:
Name: lobechat.example.top
Address: 240e:xxx
```

**Reply by BiFangKNT** [reply]
排查出了一个问题，我进入到minio容器中，连接不上我的lobe，这是怎么回事啊：
```
bash-5.1# curl -I  https://lobechat.example.top
curl: (7) Failed to connect to lobechat.example.top port 443 after 48 ms: Couldn't connect to server
```

已经映射了外部dns配置文件的：
```
bash-5.1# cat /etc/resolv.conf 
nameserver 223.5.5.5
nameserver 119.29.29.29
nameserver 8.8.8.8
nameserver 192.168.5.1
```

**Reply by BiFangKNT** [reply]
是ipv6问题。bridge网络默认关闭ipv6，而我的域名只有ipv6，因此无法解析。我现在在docker-compose配置文件中设置了
```
networks:
  lobe:
    driver: bridge
    enable_ipv6: true
    ipam:
      driver: default
```
并且给每个容器都加上网络配置：
```
    networks:
      - lobe
```
问题就迎刃而解了。
